auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -181,6 +181,14 @@
         </div>
       </div>
 
+      <div>
```

Comment:
Looks really clean! But do you need this div?

Reply:
removed div